### PR TITLE
Add Animals Table

### PR DESCRIPTION
**Pull request by AJ193**
### Implementations in this project

- Create a database named vet_clinic.
- Create a table named animals with the columns.
- Insert the data.
- Write queries with quests.
process screenshot
project 1 screenshot
Capture1
Capture2

### CODE REVIEW

File: queries.sql
Comment (AbelG101) [inline queries.sql:1]: - [x] As part of the project requirements please make sure to add the proper `animals` schema with all the columns, this is how your schema currently looks:
   image
- Project requirements:
  image

Comment (AbelG101) [inline queries.sql:1]: - [x] Once again pelase make sure to Insert the appropriate animals data as specified in the project requirements, this is how your `data.sql` currently looks:
  image

- Project requirements:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @AJ193 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct queries for each question✔
- Correct use of Git Flow ✔
- Descriptive PR title with a good summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql
Comment (AJ193) [reply queries.sql:1]: @AbelG101 Thanks for guiding i made a mistake that is why this happened.

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * from animals WHERE neutered = TRUE AND escape_attempts < 3;
```
Comment (OyePriscilla) [inline queries.sql:5]: - [x] Kindly add the screenshot of the result of these three queries to the PR as expected. ✔️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @AJ193 👍 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Create tables in SQL ✔️ 
Inserted and queried data in SQL ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (AJ193) [reply queries.sql:5]: @OyePriscilla Ok Done

**Review by Strangeal — APPROVED** [review]
Hi @AJ193, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**